packages/protocol/contracts/governance/Governance.sol

```diff
@@ -1264,4 +1387,39 @@ contract Governance is
     }
     return threshold;
   }
+
+  /**
+   * @notice Returns the total number of votes cast by an account.
+   * @param account The address of the account.
+   * @return The total number of votes cast by an account.
+   */
+  function getAmountOfGoldUsedForVoting(address account) public view returns (uint256) {
+    uint256 total = 0;
+    Voter storage voter = voters[account];
+    for (uint256 index = 0; index < dequeued.length; index = index.add(1)) {
+      VoteRecord storage voteRecord = voter.referendumVotes[index];
+      uint256 subTotal = 0;
+      for (uint256 i = 0; i < voteRecord.weights.length; i = i.add(1)) {
+        subTotal = subTotal.add(voteRecord.weights[i]);
+      }
+      if (total < subTotal) {
+        total = subTotal;
+      }
+    }
+    return total;
+  }
+
+  /**
+   * @notice Returns sum of input weights.
+   * @param weights The weights to sum up.
+   * @return The sum of input weights.
+   */
+  function getTotalWeightRequested(uint256[] memory weights) private pure returns (uint256) {
+    uint256 totalVotesRequested = 0;
+    for (uint256 i = 0; i < weights.length; i = i.add(1)) {
+      totalVotesRequested = totalVotesRequested.add(weights[i]);
```

Comment:
Can this overflow? If so, someone could vote so much that their votes overflow and thus being able to vote with (way) more voting than what it actually has.

If not possible to overflow then I'd add a comment for the future (and auditors)